winui/Ribbon/RibbonCompactSizing.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Ribbon Compact Sizing in WinUI Ribbon control | Syncfusion
+description: Learn here all about Ribbon Compact Sizing feature of Syncfusion WinUI Ribbon(SfRibbon) control with custom support and more.
```

Comment:
Learn here all about the Ribbon Compact Sizing feature of Syncfusion WinUI Ribbon (SfRibbon) control with custom support and more.

Reply:
Done the changes